pages/[lng]/blog/[slug].tsx

```diff
@@ -1,7 +1,14 @@
 /* library package */
 import { FC, useState } from 'react';
 import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
-import { BlogSingle, BlogCategories, useI18n, BlogRecent, getBlogHeaderImage } from '@sirclo/nexus'
+import {
+  BlogSingle,
+  BlogCategories,
+  useI18n,
+  BlogRecent,
+  getBlogHeaderImage,
+  useAuthToken
+} from '@sirclo/nexus';
 import Link from 'next/link';
 /* library template */
```

Comment:
```suggestion
  useAuthToken
} from '@sirclo/nexus'
import Link from 'next/link'
/* library template */
```

titik koma dihilangken